src/test/java/com/firebase/geofire/GeoQueryDataEventTestListener.java

```diff
@@ -74,7 +74,6 @@ public void onGeoQueryReady() {
 
   @Override
   public void onGeoQueryError(DatabaseError error) {
-
+    throw error.toException();
```

Comment:
I've noticed that this is using 2 spaces which is part of the Java Google Style Guide. I also prefer 2 spaces. Should we change it everywhere or was this a mistake?

Reply:
Was probably a mistake, I prefer 4 spaces but was using a new version of IntelliJ I had not set up.